src/components/drawer/index.js

```diff
@@ -225,17 +259,17 @@ class NewDrawer extends PureBaseComponent {
   renderActions(items, progress, dragX) {
     if (items) {
       return (
-        <View row>
+        <Animated.View style={[styles.actionsContainer, {transform: [{translateX: this.state.leftX}]}]}>
```

Comment:
No need to replace View with Animated.View anymore. 
You can just render 
```
<View row animated>
```

the `animated` prop will make sure to use an `Animated.View`

Reply:
Done